SD Card Structure/Compound/sys/position_movetoneutral.g

```diff
@@ -0,0 +1,6 @@
+;_________Move the print head to a central position, and the bed to the bottom________
+
+G90 ; Ensure we're absolute
+
+; Move to a position off of all axis
+G0 X200 Y200 Z380 S1
```

Comment:
If we are moving bed to bottom, lets not assume that is Z380. sounds like a good way to skip or screw something up. It would be better just to call G28 Z so it always goes to the bottom. especially since the "bottom" Z value may change depending on mods.

Reply:
The S1 at the end says to move to 380 until you hit the sensor. 380 is just an arbitrary value that *should* ensure enough movement to reach bottom regardless of where you're at.

Reply:
Re: Homing:

Homing Z instead of moving down is definitely an option. It effectively does the same thing, with the exception of the G0 only moving down, while the Home will "bounce" slightly at the bottom as it hits the switch, comes back up, then goes down until it touches it again. It's largely a matter of personal preferences, but if Homing is what everyone feels is best, I have no issues with it.

Reply:
Now updated to call homez